script/openqa-clone-custom-git-refspec

```diff
@@ -56,6 +56,10 @@ if [[ -z "$branch" ]] || [[ -z "$repo_name" ]]; then
     repo_name="${repo_name:-"${label%:*}/${target_repo_part##*/}"}"
     branch="${branch:-"${label##*:}"}"
     repo="${repo:-"https://github.com/${repo_name}.git"}"
+
+    # Pick a test to run from the PR description
+    body=$(echo "$pr_content" | jq -r '.body') || throw_json_error "$pr_url" "$pr_content"
+    job=$(echo "$body" | grep http | sed 's/http/\nhttp/g' | grep ^http | sed 's/\(^http[^ <]*\)\(.*\)/\1/g' | grep /tests/ | sort -u) || true
```

Comment:
I thought we changed that already to set "job_list"? Btw, shellcheck probably complains about this. I have shellcheck hooked into my vim and it probably tells about quotes around `$(…)`. But also: If it works it's ok but I guess we can simplify this with more jq. Also originally my idea was to have some kind of identifiers. E.g. try:

```
curl -s https://api.github.com/repos/os-autoinst/os-autoinst-distri-opensuse/pulls/6529 \
  | jq -r '.body | sub("^(?<marker>@openqa: Clone )(?<url>http.*[0-9]*)"; "\(.url)")'
```

should return just `https://openqa.opensuse.org/tests/828144`

Reply:
For the record, this was the code from before our conversation (I didn't wanna push the proof of concept which didn't actually work yet). I adapted the code to use the `job_list`, to look for the marker, and the tests cover that accordingly.

Talking about this work, I'm actually thinking about extending this to support variables as well. Although I'd propose that in another branch once we agree on the initial support.